Graphing enhancements and Treeview data validation

natalia-miller opened the pull request.

natalia-miller committed:
Multi-Select on Treeview

Added multi-select without needing to use shift or ctrl on the measure selection treeview

natalia-miller committed:
Reconnect measure dropdowns for graphing

Reconnected measure dropdown on DashboardPage with MeasureSelectionPage and added a check to ensure only measures with valid results are populated

natalia-miller committed:
Rewrite measure_name_map 

Renamed measure_name_map to measure_requirements and added requirements for each statistical measure

natalia-miller committed:
Add scipy to requirements.txt

natalia-miller committed:
Update get_compatible_measures

Updated get_compatible_measures to check measure_requirements

natalia-miller committed:
Update statisical measure treeview

A little wonky right now. But the statistical measure treeview will now populate only with the applicable measures. Currently, "Show all measures" must be checked/unchecked in order for treeview to refresh . . .

natalia-miller committed:
Add measure_supports_grouping function

Checks if measure supports grouping or not for graphing column dropdown

natalia-miller committed:
Make groupby column dropdown conditional

jarrettcmiller committed:
Added "all_select_events" binding for populate_treeview. Removed the destroying of the table in the table controller update_table on CSV import. The table instance on the measurement selection page will remain the same throughout.

natalia-miller committed:
Update main_controller.py

natalia-miller committed:
Update statisticsLogic.py

natalia-miller committed:
Update GUI.py

natalia-miller committed:
Fixed "Please select No Grouping" 

Fixed "Please select No Grouping" showing up for measures that are no eligible for grouping

natalia-miller committed:
Add additional Chi Square checks to get_compatible_measures

natalia-miller committed:
Add sum check error handling to Chi Squared

natalia-miller committed:
Retrieve binomial distribution parameters

natalia-miller committed:
Retrieve binomial distribution parameters

natalia-miller committed:
Fix coefficient of variation

natalia-miller committed:
Fix coefficient of variation

natalia-miller committed:
Correlation Coefficient Graphing - GUI.py

natalia-miller committed:
Correlation Coefficient Graphing - main_controller.py

natalia-miller committed:
Update column count rules for measure validation

natalia-miller committed:
Fix Spearman Scatterplot

sarahibsen committed:
manually inputted data is able to be computed on

sarahibsen committed:
Percentiles fixed

sarahibsen committed:
Variance calculated / chosen for user based on data set

sarahibsen commented:
- [x] [ ] Sign test
- [x] [ ] probability distribution
- [x] [ ] merge conflicts 
- [x] add stat measure to work

sarahibsen committed:
merge conflicts fixed

sarahibsen committed:
allowed user to create their own statistic

sarahibsen committed:
chi square drop down taken out. sign test drop down put in

natalia-miller committed:
Add default graphs for custom measures

natalia-miller committed:
Add default graphs for custom measures

jarrettcmiller committed:
Updated format_results to check if the key in dictionary matches the measure name, so the result popup doesn't duplicate.

natalia-miller committed:
Change Correlation to Correlation Coefficient for graphing

natalia-miller committed:
Fix correlation coefficient

natalia-miller committed:
Fix Mode data error

Change return None to raise TypeError so measure is detected as incompatible

natalia-miller committed:
Add Mode to must be grouped and fix custom measure error

natalia-miller committed:
Remove bar chart from binomial distribution

natalia-miller committed:
Fix custom measure detection

Also removed correlation from bar chart since it is no longer needed there.

natalia-miller committed:
Fix custom measure detection for grouping

sarahibsen committed:
Placement of the inputs for each measure now responsive

sarahibsen committed:
Merge branch 'Graphing-Enhancements' of https://github.com/sarahibsen/CS499 into Graphing-Enhancements

sarahibsen committed:
had to fix results pop up. also recomipiled exe

sarahibsen committed:
a more readable txt export

sarahibsen committed:
took out env

sarahibsen committed:
took out env

sarahibsen committed:
drop string value data types

jarrettcmiller committed:
Trimmed home_listings to 20 homes. Updated results label for Chi-Squared, Sign Test, Rank Sum, and Spearman Correlation

sarahibsen committed:
Spearman pop up. cleaned up many functions. one centralized data util function

natalia-miller committed:
Fix Spearman Rank Correlation Graphing

natalia-miller committed:
Fix Spearman Rank Correlation Graphing

jarrettcmiller committed:
Fixed issue with Chi-Squared where column headers were a mixer of user defined and default (A-Z).

natalia-miller committed:
Merge pull request #117 from sarahibsen/113-clean

113 clean

natalia-miller committed:
Merge branch 'Graphing-Enhancements' into results-popup-duplicate-measures

natalia-miller committed:
Merge pull request #118 from sarahibsen/results-popup-duplicate-measures

Results popup duplicate measures

natalia-miller committed:
Fix pie chart and graph titles

natalia-miller committed:
Add Pie Chart to Mean & Median

natalia-miller committed:
Remove duplicate get_compatible_measures function

natalia-miller committed:
Fix pie chart . . . again :D

Also restored mean and median grouping because I broke it in the last commit . . .

natalia-miller committed:
PR 116 Fix signtest - Update statisticsLogic.py

natalia-miller committed:
Remove sign test from grouped data logic

natalia-miller committed:
Change Sign Test to No Grouping

natalia-miller committed:
Fix ravel error for sign test

natalia-miller committed:
Add unique values check for mode

natalia-miller committed:
Add rho and p to Spearman
